docs/components/integration-framework/connectors/out-of-the-box-connectors/available-connectors-overview.md

```diff
@@ -9,5 +9,6 @@ This section gives an overview of the **out-of-the-box connectors** available in
 - [REST connector](rest.md) - Make a request to a REST API and use the response in the next steps of your process.
 - [SendGrid connector](sendgrid.md) - Quickly send emails from your BPMN processes.
 - [Slack connector](slack.md) - Send messages to channels or users in your [Slack](https://slack.com) workspace from your BPMN process.
+- [AWS SQS connector](aws-sqs.md) - Send messages to AWS [Simple Queue Service](https://aws.amazon.com/sqs/) from your BPMN process.
```

Comment:
Since our list grows and grows, let's sort it alphabetically

Reply:
Done